ev3dev2/__init__.py

```diff
@@ -199,8 +199,8 @@ def __repr__(self):
     def _attribute_file_open(self, name):
         path = os.path.join(self._path, name)
         mode = stat.S_IMODE(os.stat(path)[stat.ST_MODE])
-        r_ok = mode & stat.S_IRGRP
-        w_ok = mode & stat.S_IWGRP
+        r_ok = mode & stat.S_IRUSR
+        w_ok = mode & stat.S_IWUSR
```

Comment:
I am not sure now, but I think group permission are checked for a reason. Attributes are root-owned, and to read some of them, we add users to a specific groups. See f49158b93aef7eb312de7da5f45a7606384e1fbe and #84

Reply:
Can we alter group permissions on the CI server before running the tests?

Reply:
Hmph, I was wondering about that. We probably can add such logic to the `fake-sys` script as long as we know what to add... I assume it's a `chmod` call with these flags added but I don't fully understand the Unix permissions model so I'd have to test it.